src/backend/web-gl/function-node.js

```diff
@@ -848,93 +876,143 @@ module.exports = class WebGLFunctionNode extends FunctionNodeBase {
 							retArr.push('[');
 							retArr.push(mNode.property.raw);
 							retArr.push(']');
+							multiMemberExpression && this.popState('not-in-get-call-parameters');
```

Comment:
Can you write this less cryptically?

Reply:
you mean:
if (multiMemberExpression) {
  this.popState('not-in-get-call-parameters');
}

pretty standard in JS! kinda powerful...but happy to rewrite.  You got a coding standard?

I think this is an eslint option you can turn on/off - i think AirB&B eslint standard has it off which has annoyed be before!

Reply:
```js
if (multiMemberExpression) {
  this.popState('not-in-get-call-parameters');
}
```

Yes, this would be more straightforward to the next human and is the existing standard in the library thus far.